wasm/nodejs/simple-transaction.js

```diff
@@ -53,11 +53,17 @@ async function runDemo() {
         // a very basic JS-driven utxo entry sort
         entries.sort((a, b) => a.utxoEntry.amount > b.utxoEntry.amount || -(a.utxoEntry.amount < b.utxoEntry.amount));
 
+        let total = entries.reduce((agg, curr) => {
+            return curr.utxoEntry.amount + agg;
+        }, 0n);
+
+        console.info('Amount sending', total - BigInt(entries.length) * 2000n)
+
         let { transactions, summary } = await createTransactions({
-            entries,
-            outputs : [[destination, kaspaToSompi(0.2)]],
+            entries, 
+            outputs : [[destinationAddress, total - BigInt(entries.length) * 2000n]],
             priorityFee: 0,
-            changeAddress: address,
+            changeAddress: destinationAddress,
```

Comment:
`changeAddress` should be `skAddress`...   otherwise you are relaying all funds to the destination.

Also, is the `total` intentional?  I mean I see it is, but given that there is the same keypair in the demo on testnet this will just drain all the funds...  Why not set this to `0.2` and let whoever testing experiment with it...  like this prevents me from even testing this script multiple times as it will just drain whatever I send to this address...

otherwise LGTM

Reply:
Changed to send only `0.0012`